integration-tests/artifacts/gatsby-browser.js

```diff
@@ -0,0 +1,9 @@
+const React = require(`react`)
+const Github = require(`./src/components/github`).default
+
+// exports.wrapPageElement = ({ element }) => (
+//   <>
+//     <Github />
+//     {element}
+//   </>
+// )
```

Comment:
is this file doing anything? should we remove it?

Reply:
It is, I forgot to uncomment the stuff but the fact that those imports are present make it so that the queries are included in every page's page-data (which the integration tests assert)